package.json

```diff
@@ -59,5 +59,8 @@
     "javascript",
     "webgl",
     "browser"
-  ]
+  ],
+  "publishConfig": {
```

Comment:
I take it this cannot be done from the command line, since we're relying on Travis do execute the npm publish. Is that correct?